src/model/CplacePlugin.ts

```diff
@@ -50,6 +52,7 @@ export default class CplacePlugin {
         this.repo = path.basename(path.dirname(path.resolve(pluginDir)));
         this.assetsDir = path.resolve(pluginDir, 'assets');
         this.hasTypeScriptAssets = fs.existsSync(path.resolve(this.assetsDir, 'ts'));
+        this.hasTypeScriptE2EAssets = fs.existsSync(path.resolve(this.assetsDir, 'e2e/specs'));
```

Comment:
Do not use the `/` directly - just use two arguments: `this.assetsDir, 'e2e', 'specs'